### PR TITLE
Some edits for shared project embedding

### PR DESCRIPTION
**Pull request by ganicke**
A few edits for the `blocks-embed` page.

RE: #6262

### CODE REVIEW

File: common-docs/blocks-embed.md [hunk 1]
```diff
@@ -349,6 +349,6 @@ f.contentWindow.postMessage({
 
 * [HTML only](https://jsfiddle.net/L8msdjpu/3/)
```
Comment (ganicke) [inline common-docs/blocks-embed.md]: What does "HTML only" mean? - "An HTML render example of an embedded project"?

File: common-docs/blocks-embed.md [hunk 2]
```diff
@@ -325,22 +325,22 @@ $(function () {
...
 
-Then, read the data-attribute and pass it in the ``renderblocks`` call.
+Then, read the ``data-packageid`` attribute and pass it along as the ``packageId`` field in the ``options`` of the ``renderblocks`` message.
 
 ```
 f.contentWindow.postMessage({
     type: "renderblocks",
     id: pre.id,
-    code: pre.innerText,
+    code: "",
```
Comment (ganicke) [inline common-docs/blocks-embed.md]: Leave the `code` attribute blank?

Comment (pelikhan) [reply common-docs/blocks-embed.md]: Yes by design

File: common-docs/blocks-embed.md (hunk 1 above)
Comment (pelikhan) [reply common-docs/blocks-embed.md]: Meaning a basic example without using any kind of JS framework

**Pull request merged**